src/pages/Payment.vue

```diff
@@ -1,11 +1,58 @@
 <template>
     <div>
-        <main class="container" id="payment">
-
-            <div class="content">
+        <main class="container fullwidth" id="payment">
 
+            <div class="content fullwidth">
                 <h1>My Voucher List</h1>
 
+                <p>You can request payment for <strong>3</strong> vouchers.</p>
+
+                <div class="voucher-list">
+
+                    <!-- Tab header -->
+                    <div class="tab thead">
+                        <label for="tab-one">
+                            <div class="row">
+                                <div>Voucher code</div>
```

Comment:
Can you sort this off fixtures before we merge in? Or you need it in now for style tweaking?